js/menu.js

```diff
@@ -29,4 +52,8 @@ function init() {
       action: 'about-overlay'
     });
   });
+}
+
+if (menuInstanceId) {
```

Comment:
not sure why this has been moved down here, feels like an unnecessary change.

Reply:
It is unnecessary. It's moved to follow our standard _variables-functions-then-code_ flow. That's all.